scout/server/blueprints/panels/views.py

```diff
@@ -100,9 +111,15 @@ def panels():
     )
 
 
+def shall_display_panel(panel_obj, user):
```

Comment:
I'm not really sure why codefactor complains here.. Perhaps rename into `display_panel_status` and move it into controllers, since it's not strictly an endpoint? But it's up to you!